testproject/Assets/Tests/Runtime/AddressablesTests.cs

```diff
@@ -0,0 +1,310 @@
+using System.Collections;
```

Comment:
This needs to be wrapped in #if NETCODE_USE_ADDRESSABLES or it will not compile if the addessables package is not included in the project.

Reply:
Is there a reason we'd remove addressables from testproject?

Reply:
Well, this sort of brings up a broader topic about how we can test with and without addressables to make sure no future changes break the SDK when compiled without the addressables package included.  Since it is now part of our primary testproject, all of our tests will be run against NETCODE_USE_ADDRESSABLES defined builds.  We will need to most likely add a new undefined NETCODE_USE_ADDRESSABLES build in the nightly builds in order to run our tests against that.
So, we will need to be able to compile our test project when the addressable package is not included in order to be able to do this.